atlp-rwanda/ecommerce-app-predators-fe
# #51 #47 chore: tests-ci-cd

#### What does this PR do?
- Configure testing with jest library
- Connect repository with codeclimate, and display maintainability badge after successful test run.
#### Description of Task to be completed

> We are going to use Jest to write unit tests for our components in react as well as integration tests for the various pages on our client. Setup the Jest library and add any commands required for testing in the README of the project.

> Run Codeclimate test coverage after passing the tests from Github Actions. We should be able to see the maintainability badge after successful test run with CI

#### How should this be manually tested?
- Clone the repository to your local pc
- Checkout ch-test-ci-cd  branch
- Write any tests in the src/components/\_\_tests\_\_  folder
- Run npm run test , this should run all tests and generate a coverage report
#### What are the relevant pivotal tracker stories?
- Cards [#51](https://trello.com/c/8y2mlAv5) [#47](https://trello.com/c/DuD2CSVg)
